doc/nrf/releases_and_maturity/migration/nRF54H20_migration_2.7/transition_guide_2.4.99-cs3_to_2.7_environment.rst

```diff
@@ -0,0 +1,295 @@
+         1. With admin permissions enabled, download and install the `nRF Command Line Tools`_.
+         #. Restart the Toolchain Manager application.
+         #. Click the dropdown menu for the installed nRF Connect SDK version.
+
+            .. figure:: ../../../../nrf/installation/images/gs-assistant_tm_dropdown.png
+               :alt: The Toolchain Manager dropdown menu for the installed nRF Connect SDK version, cropped
+
+               The Toolchain Manager dropdown menu options
+
+         #. Select :guilabel:`Open command prompt`.
+
+         You can then follow the instructions in :ref:`creating_cmd`.
+
+#. If you have any existing custom applications created for 2.4.99-cs3 that you would like to migrate, move its files from the previous ``ncs-lcs_old`` folder to the newly created |NCS| installation folder.
+
+Updating the Terminal application
+*********************************
+
+To update `Serial Terminal from nRF Connect for Desktop`, follow these steps:
+
+1. On your computer, open `nRF Connect for Desktop`_
+   If there is an update available, a pop up will notify you of its availability.
+#. If available, install the update from the pop up screen.
+#. Update `Serial Terminal from nRF Connect for Desktop`.
+
+If you are using the nRF Terminal application part of the `nRF Connect for Visual Studio Code`_ extension, open Visual Studio Code instead and ensure you are running the newest version of both the editor and the extension.
+
+.. _migration_cs3_to_2_7_env_bringup:
+
+nRF54H20 DK bring-up
+********************
+
+The following sections describe the steps required for the nRF54H20 bring-up.
+
+.. rst-class:: numbered-step
+
+Programming the BICR
+====================
+
+..
+   ##TODO update BICR file link
+
+The Board Information Configuration Registers (BICR) are non-volatile memory (NVM) registers that contain information on how the nRF54H20 SoC must interact with other board elements, including the information about the power and clock delivery to the SoC.
+To prepare the nRF54H20 DK for first use, you must manually program the values of the BICR using a precompiled BICR binary file (:file:`bicr_ext_loadcap.hex`).
+
+1. Download the `BICR binary file`_ .
+#. List all the connected development kits to see their serial number (matching the one on the DK's sticker)::
+
+      nrfutil device list
+
+#. Move the BICR HEX file to a folder of your choice, then program the BICR by running nRF Util from that folder using the following command::
+
+      nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr_ext_loadcap.hex --core Secure --serial-number <serial_number>
+
+.. rst-class:: numbered-step
+
+Programming the SDFW and SCFW
+=============================
+
+..
+   ##TODO update binaries version
+
+After programming the BICR, the nRF54H20 SoC requires the provisioning of a bundle ( :file:`nrf54h20_soc_binaries_v0.3.3.zip`) containing the precompiled firmware for the Secure Domain and System Controller.
```

Comment:
nrf54h20_soc_binaries_v0.3.3.zip Hmm I not sure if this is valid version

Reply:
That needs to be updated when 0.5.0 bundle will be officially out